### PR TITLE
Allow og:image meta tag to be customisable

### PR DESCRIPTION
**Pull request by igloosi**
Moving the meta tag into the `head` block so that it can be overwritten, as the image path for it needs to be an absolute path.

Fixes: #813

### CODE REVIEW

File: src/template.njk [hunk 1]
```diff
@@ -18,10 +18,11 @@
       <link rel="apple-touch-icon" href="{{ assetPath | default('/assets') }}/images/govuk-apple-touch-icon.png">
     {% endblock %}
 
-    {% block head %}{% endblock %}
-
+    {% block head %}
```
Comment (nickcolley) [inline src/template.njk]: This is a breaking change since before even when you set head you'd get the default open graph image.

Could this be solved by something like

```html
<meta property="og:image" content="{{ opengraphAssetPath | default('') }}{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
```

Comment (igloosi) [reply src/template.njk]: tru dat. updated so that the variable can just be set

File: src/template.njk [hunk 2]
```diff
@@ -19,9 +19,9 @@
     {% endblock %}
 
     {% block head %}{% endblock %}
-
     {# The default og:image is added below head so that scrapers see any custom metatags first, and this is just a fallback #}
-    <meta property="og:image" content="{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
+    {# image path needs to absolute e.g. http://wwww.domain.com/.../govuk-opengraph-image.png #}
```
Comment (36degrees) [inline src/template.njk]: Path refers to just the part after the domain - i.e. just the `/.../govuk-opengraph-image.png` bit. How about `assetUrl`?

Comment (igloosi) [reply src/template.njk]: sure. updated

**Comment by nickcolley** [comment]
Would you mind raising an issue after this is merged on the Design System repo to include this new feature in the guidance?

File: src/template.njk [hunk 3]
```diff
@@ -19,9 +19,9 @@
     {% endblock %}
 
     {% block head %}{% endblock %}
-
     {# The default og:image is added below head so that scrapers see any custom metatags first, and this is just a fallback #}
-    <meta property="og:image" content="{{ assetPath | default('/assets') }}/images/govuk-opengraph-image.png">
+    {# image url needs to be absolute e.g. http://wwww.domain.com/.../govuk-opengraph-image.png #}
+    <meta property="og:image" content="{{ assetUrl | default('/assets') }}/images/govuk-opengraph-image.png">
```
Comment (nickcolley) [inline src/template.njk]: I think the default needs to be 'assetPath' otherwise this is a breaking change still?

Comment (nickcolley) [reply src/template.njk]: maybe `{{ openGraphImagePath | assertPath | default('/assets') }}`

not sure

Comment (igloosi) [reply src/template.njk]: it doesn't work with assetPath anyway, so not sure it's a breaking change

Comment (nickcolley) [reply src/template.njk]: Fair point, but consider if someone set their 'assetPath' to a absolute path, it would be working for those users?

Comment (igloosi) [reply src/template.njk]: modified so that then default value for assetUrl is assetPath, insuring there it is not a breaking change if anyone specified an absolute url for the assets

File: CHANGELOG.md [hunk 4]
```diff
@@ -28,6 +28,15 @@
   ([PR #837](https://github.com/alphagov/govuk-frontend/pull/837) and
    [PR #848](https://github.com/alphagov/govuk-frontend/pull/848))
 
+- Use relative line-height
```
Comment (36degrees) [inline CHANGELOG.md]: This shouldn't be part of this PR.

File: CHANGELOG.md [hunk 5]
```diff
@@ -28,6 +28,15 @@
...
+  Update typography styles to use relative, unitless line-height
+  ([PR #837](https://github.com/alphagov/govuk-frontend/pull/837))
+
+- Allow og:image meta tag to be customisable
```
Comment (36degrees) [inline CHANGELOG.md]: It was already customisable by modifying `assetPath` – I think we need to be clearer as to what the change is here.

**Comment by nickcolley** [comment]
Original code: https://github.com/alphagov/govuk_template/blob/master/source/views/layouts/govuk_template.html.erb#L36

**Comment by igloosi** [comment]
i've updated the wording

**Comment by nickcolley** [comment]
Thinking about this more, if this is solved by setting 'assetPath' to an absolute url do we need to do this at all?

**Comment by igloosi** [comment]
we can't set the absolute path for example in the design system (as we have two different urls)
but if thats not of a concern and we just leave the declaration twice as we have now, then there is no need for this indeed
